index.html

```diff
@@ -16,30 +16,30 @@
         <h1>All awsome books</h1>
       </div>

```

Comment:
- [ ] Let's correct this typo here. It is `awesome` and not `awsome` 😉.

- [ ] I am adding this comment here because you might not see it in the summary if I added it there. Why not add a descriptive pull request description that covers what you implemented in this milestone. This will make the pull request more descriptive and more professional

- [ ] I am adding this comment here because I can't find the README file in the files changed. Let's improve the README by adding instructions on how to clone your repository and get it working in the local machine. Let's also remove some of the instructions that are not yet implemented in this project and the text generated from the README template like the ones in the image below by replacing them with your instructions.

<img width="474" alt="Loom_TOpH7thwlB" src="https://user-images.githubusercontent.com/81468675/153092841-8c0b4ff9-32cd-4ec0-9798-0744c289b265.png">